app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -4,6 +4,18 @@ module ApplicationController::MiqRequestMethods
     helper_method :dialog_partial_for_workflow
   end
 
+  def get_provisioning_request_source(src_id, src_type_string)
```

Comment:
@mansam - It would be nice to DRY up the behavior in this method. I noticed a few methods that duplicated the behavior.